build/settings.targets

```diff
@@ -19,6 +19,7 @@
         <Otherwise>
```

Comment:
The lines above this sign the axe-windows CLI project with a different (1st party) cert. I believe this should be left as-is, but let me know if that doesn't seem right.

Reply:
The MSI properties list Microsoft as the publisher, so I think this would make sense. @RobGallo or @peterdur might have more recent insights based on the thread, but that could certainly be in a separate PR, if needed.